hello-world-ruby-webapp/README.md

```diff
@@ -0,0 +1,9 @@
+# Hello World Ruby
+
+Sample for Hello World web app.
+
+## Getting started
+
+1. Fork this repository.
+2. Delpoy as a web app in Choreo. 
```

Comment:
Shall we add more details?

Reply:
Fixed with https://github.com/wso2/choreo-samples/pull/15/commits/add09046e6b8c2d071d9af8ab6d9b1722a30038e